src/providers/MeetingProvider/docs/MeetingManager.stories.mdx

```diff
@@ -131,7 +129,51 @@ const MyApp = () => {
 }
 ```
 
-2. Share the same `MeetingManager` instance with multiple different `MeetingProvider`s.
+### Update the `MeetingSessionConfiguration` properties being passed to the `join` method
+
+With V3 of the SDK, builders now have the option of updating the properties of Meeting Session Configuration. Meeting Session Configuration is a Chime JS SDK class that allows the users to configure the meeting session.
+
+The class contains several properties like `videoUplinkBandwidthPolicy`, `videoDownlinkBandwidthPolicy`, `attendeePresenceTimeoutMs` etc. You can learn more about the available properties at [MeetingSessionConfiguration](https://aws.github.io/amazon-chime-sdk-js/classes/meetingsessionconfiguration.html)
+in the Chime JS SDK documentation. The following code example shows a meeting session configuration where the default value of the properties are overridden to cusomize the meeting session.
+
+```jsx
+import { useMeetingManager } from 'amazon-chime-sdk-component-library-react';
+import { MeetingSessionConfiguration, VideoPriorityBasedPolicyConfig } from 'amazon-chime-sdk-js';
+
+const MyApp = () => {
+  const meetingManager = useMeetingManager();
+
+  const joinMeeting = async () => {
+    // Fetch the meeting and attendee data from your server application
+    const response = await fetch('/my-server');
+    const data = await response.json();
+
+    // Initalize the meeting session configuration
+    let meetingSessionConfiguration = new MeetingSessionConfiguration(data.JoinInfo.Meeting, data.JoinInfo.Attendee);
+    // Update the meeting session configuration properties
+    meetingSessionConfiguration.attendeePresenceTimeoutMs = 120;
+    const config = VideoPriorityBasedPolicyConfig.UnstableNetworkPreset;
```

Comment:
nit: need to rename this `config` to make it more specific.
suggest remove this statement and use it directly:

```js
meetingSessionConfiguration.videoDownlinkBandwidthPolicy = new VideoPriorityBasedPolicy(
  logger,
  VideoPriorityBasedPolicyConfig.UnstableNetworkPreset,
);
```

Reply:
Done